bin/install-php-phpunit.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+if [[ ${SWITCH_TO_PHP:0:3} == "5.2" ]] || [[ ${SWITCH_TO_PHP:0:3} == "5.3" ]]; then
+  # install php runtime dependencies
+  sudo apt-get install -y curl libxml2 libfreetype6 libpng12-0 libjpeg8 libgd3 libxpm4 \
+  libltdl7 openssl gettext libgettextpo0 libmhash2 libmcrypt4 mysql-client
+
+  if [[ ${SWITCH_TO_PHP:0:3} == "5.2" ]]; then
+    PHPBREW_BUILT_CHECK=$HOME/.phpbrew/php/php-5.2.17/bin/php
+  else
+    PHPBREW_BUILT_CHECK=$HOME/.phpbrew/php/php-5.3.29/bin/php
+  fi
+
+  # php and phpunit installs should be cached, only build if they're not there.
+  if [ ! -f $PHPBREW_BUILT_CHECK ]; then
+    
+    # install build dependencies for building php (yes, for phpbrew, php is a dependency)
+    sudo apt-get install -y php5-dev autoconf automake curl libcurl3-openssl-dev build-essential \
+    libxslt1-dev re2c libxml2-dev php5-cli bison libbz2-dev libreadline-dev libfreetype6-dev \
+    libpng12-dev libjpeg-dev libjpeg8-dev libgd-dev libltdl-dev libssl-dev libgettextpo-dev \
+    libicu-dev libmhash-dev libmcrypt-dev libmysqlclient-dev libmysqld-dev git
+
+    # install phpbrew for this user
+    mkdir $HOME/php-utils-bin
+    cd $HOME/php-utils-bin
+    curl -L -O https://github.com/phpbrew/phpbrew/raw/master/phpbrew
```

Comment:
Yes, I'm just making sure that it isn't breaking every other test first ;) (which it was)